server/filters/formatCuriousCourseStatusFilter.ts

```diff
@@ -0,0 +1,11 @@
+export default function formatCuriousCourseStatusFilter(value: string): string {
+  const curiousCourseStatusValue = CuriousCourseStatusValues[value as keyof typeof CuriousCourseStatusValues]
+  return curiousCourseStatusValue
+}
+
+enum CuriousCourseStatusValues {
```

Comment:
New filter for the Curious course status values. It is used on the Withdrawn course tab for differentiating between Withdrawn and Temporarily Withdrawn; but we could use the filter on the other tabs if the design changes